collections/crowdsecurity/ftp.md

```diff
@@ -0,0 +1 @@
+A collection to defend FTP against bruteforce attack.
```

Comment:
I think in the collection we need to say what logs we support, and give a bit more info about the bruteforce (ie. speed / capacity ?)

Reply:
especially if it's **specific** to VSFTPD we need to tell it